doc/source/user_guide/getting_started.rst

```diff
@@ -41,3 +41,18 @@ from image files, using :func:`skimage.io.imread`: ::
     >>> from skimage import io
     >>> moon = io.imread(filename)
 
+Use natsort to load multiple images: ::
+
+    >>> import os
+    >>> from natsort import natsorted, ns
+    >>> from skimage import io
+    >>> list_files = os.listdir('.')
+    >>> list_files
+    ['01.png', '010.png', '0101.png', '0190.png', '02.png']
+    >>> list_files = natsorted(list_files, alg=ns.PATH)
```

Comment:
`alg=ns.PATH` seems to be redundant:
```
In [4]: list_files = ['01.png', '010.png', '0101.png', '0190.png', '02.png']
In [5]: list_files = natsort.natsorted(list_files)                              

In [6]: list_files                                                              
Out[6]: ['01.png', '02.png', '010.png', '0101.png', '0190.png']
```

Reply:
I agree with this. ns.PATH _does_ seem to be suitable for edge cases as defined [here](https://natsort.readthedocs.io/en/master/examples.html#path-sort).

For most use cases, ns.PATH is redundant.